cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -2,6 +2,7 @@ use std::env;
 
 use once_cell::sync::Lazy;
 
+pub const SERVICES_ENV_VAR: &str = "FLOX_FEATURES_SERVICES";
```

Comment:
nonblocking: looks like this is unused?

Reply:
I'll clean it up in the YAML ticket, I'd like to get this in since it's blocking other tickets